source/production/F0.Minesweeper.Server/wwwroot/css/site.css

```diff
@@ -18,12 +22,12 @@ a {
 	z-index: 1000;
 }
 
-#blazor-error-ui .dismiss {
-	cursor: pointer;
-	position: absolute;
-	right: 0.75rem;
-	top: 0.5rem;
-}
+	#blazor-error-ui .dismiss {
+		cursor: pointer;
+		position: absolute;
+		right: 0.75rem;
+		top: 0.5rem;
+	}
```

Comment:
Some more formatting shenanigans? 😉

Reply:
auto formatting from _Visual Studio_ - indented because there is a "parent" `#blazor-error-ui` _id Selector_

Reply:
Found Stackoverflow pages to this "problem"/setting:
https://stackoverflow.com/questions/47094652/visual-studio-2017-disable-css-formatting-unnecessary-spaces
https://stackoverflow.com/questions/25407931/prevent-indent-when-formatting-css/25412948

Reply:
as discussed: reverted hierarchical CSS indentation